contractsdk/cpp/XRC/xrc01/src/main/xrc01.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "xchain/xchain.h"
+#include "table/types.h"
+#include "table/table.tpl.h"
+#include "pb/xrc01.pb.h"
+
+// XRC01 is the implement of XRC-01 multi token standard
+class XRC01 {
+    public: 
+        XRC01(xchain::Context* ctx): _token(ctx, "token"), _asset_info(ctx, "asset_info"), _authorize_info(ctx, "_authorize_info") {
```

Comment:
Consider following C++ code style to keep each line no longer than 80 chars.

Reply:
fix it